updatehub/tests/integration_test.rs

```diff
@@ -0,0 +1,427 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
+
+fn write_basic_config(path: PathBuf) -> String {
+    format!(
+        r#"
+[network]
+server_address="https://api.updatehub.io"
+listen_socket="localhost:8000"
+
+[storage]
+read_only=false
+runtime_settings="/data/updatehub/state.data"
```

Comment:
This should be a path to a `tempfile::NamedTempFile` so it will be cleaned up automatically.